x/concentrated-liquidity/position.go

```diff
@@ -62,6 +63,37 @@ func (k Keeper) initOrUpdatePosition(
 
 	// TODO: consider deleting position if liquidity becomes zero
 
+	// Create records for relevant uptime accumulators here.
```

Comment:
Can you explain what happens to an uptime accumulator once the position is deleted? I just want to make sure it also gets deleted from state when that happens

Reply:
Discussed on call, created issue here: #4183